Sources/docc/DocCDocumentation.docc/Reference Syntax/API Reference Syntax/Layout/Small.md

```diff
@@ -0,0 +1,32 @@
+# ``docc/Small``
+
+<!-- Directive links have different path components in source and in the generated symbol graph (where they're all top-level) -->
+<!-- Override the original documentation comment with a copy that replaces `Row/Column` with `Column` -->
```

Comment:
We could also revert this commit and live with the broken link for now. It's still pretty readable, just not clickable. https://developer.apple.com/documentation/docc/small

Reply:
I was planning on opening a separate issue to make the generated directive SGF have a hierarchy —either based on the original symbols or based on where they're supported (which would for example add a path component to all the Metadata directives)

Reply:
https://github.com/apple/swift-docc/issues/705